TrafficCapture/dockerSolution/src/main/docker/migrationConsole/cluster_tools/src/cluster_tools/base/main.py

```diff
@@ -4,41 +4,37 @@
 import argcomplete
 from console_link.environment import Environment
 import logging
-import sys
+from datetime import datetime
 
-logger = logging.getLogger(__name__)
-logger.setLevel(logging.INFO)
-
-
-class ClusterToolsFilter(logging.Filter):
-    def filter(self, record):
-        return record.name.startswith('cluster_tools') or record.name.startswith('tools')
+root_logger = logging.getLogger()
+root_logger.setLevel(logging.INFO)
```

Comment:
would be nice to support making this changeable down the line, but not a blocker for now.